packages/blade/src/tokens/theme/overrideTheme.ts

```diff
@@ -0,0 +1,23 @@
+import { cloneDeep, isEqual, merge, isPartialMatchObjectKeys, DeepPartial } from '../../utils';
+import { paymentsTheme } from '../theme';
+import type { Theme } from './theme.d';
+
+type OverrideTheme = {
+  baseTheme: Theme;
+  overrides: DeepPartial<Theme>;
+};
+
+const overrideTheme = ({ baseTheme, overrides }: OverrideTheme): Theme => {
+  if (!isEqual(baseTheme, paymentsTheme)) {
```

Comment:
it's just a base theme, take any theme it adheres to just one structure.

Reply:
okay wait my bad. I just want to check the structure. Damn how did I miss this 🤔

Reply:
Now I'm just thinking is it even worth it to have this check? Because I can't magically identify which theme to match it against. might as well remove this check?